sample/res/values/colors.xml

```diff
@@ -2,6 +2,7 @@
 <resources>
   <color name="black">#FF000000</color>
   <color name="semi_transparent_black">#7f000000</color>
+  <color name="black_transparent">#00000000</color>
```

Comment:
@pedrovgs It's black transparent. Exist a huge difference between white transparent and black transparent. It's used on gradient and influence on the colors of the element.